authorize_request_handler.go

```diff
@@ -80,5 +79,41 @@ func (c *Fosite) NewAuthorizeRequest(ctx context.Context, r *http.Request) (Auth
 
 	// Remove empty items from arrays
 	request.SetRequestedScopes(removeEmpty(strings.Split(r.Form.Get("scope"), " ")))
+
+	// Proof Key for Code Exchange by OAuth Public Clients
+	// https://tools.ietf.org/html/rfc7636
+	//
+	// PKCE is only for code response type
+	if request.ResponseTypes.Has("code") {
```

Comment:
Same thing here, this shouldn't be a hardcoded thing but instead an add-in via handlers

Reply:
Yep I'm agree. Applying "Do it, do it right, to do better" philosophy ^^